core/templates/pages/community-dashboard-page/opportunities-list-item/opportunities-list-item.directive.ts

```diff
@@ -31,7 +31,8 @@ angular.module('oppia').directive('opportunitiesListItem', [
         getOpportunity: '&opportunity',
         onClickActionButton: '=',
         isLabelRequired: '&labelRequired',
-        isProgressBarRequired: '&progressBarRequired'
+        isProgressBarRequired: '&progressBarRequired',
+        opportunityHeadingTruncationLength: '='
```

Comment:
Do we need a two-way binding here? If not I would suggest using `&`

Reply:
Done.